cupy/linalg/decomposition.py

```diff
@@ -357,7 +357,16 @@ def svd(a, full_matrices=True, compute_uv=True):
         m, n = a.shape
         x = a.transpose().astype(a_dtype, order='C', copy=True)
         trans_flag = True
+
     mn = min(m, n)
+    if mn == 0:
+        if compute_uv:
+            if full_matrices:
+                return cupy.eye(m, m), cupy.array([]), cupy.empty((0, 0))
```

Comment:
How about writing as follows for the consistency?
```suggestion
                return cupy.eye(m, m), cupy.empty((0,)), cupy.empty((0, 0))
```

Reply:
Done